dashboard/client/src/components/StatusChip.tsx

```diff
@@ -78,6 +78,7 @@ const colorMap = {
     [ServeSystemActorStatus.HEALTHY]: green,
     [ServeSystemActorStatus.UNHEALTHY]: red,
     [ServeSystemActorStatus.STARTING]: orange,
+    [ServeSystemActorStatus.INACTIVE]: blueGrey,
```

Comment:
Can we change `INACTIVE`/`ACTIVE` to `UNAVAILABLE`/`AVAILABLE`? A proxy can be unavailable but still serving pending requests, which I would consider active.

Reply:
"unavailable" indicates to me that this proxy cannot serve requests

maybe let's go with `DRAINING`? that seems more future-proof...